app/experimenter/experiments/tests/test_serializers.py

```diff
@@ -373,12 +373,22 @@ def test_serializer_excludes_countries_if_none_set(self):
 class TestCloneSerializer(MockRequestMixin, TestCase):
 
     def test_clone_serializer_rejects_duplicate_name(self):
```

Comment:
The test is called 'rejects duplicate name', but it's testing two things in one test, duplicate names and duplicate slugs.  Should those be two separate tests......?

Reply:
i guess